pkgs/applications/editors/marker/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+, gtk3
+, gtksourceview
+, gtkspell3
+, webkitgtk
+, pandoc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "marker";
+  version = "2020.04.04";
+
+  src = fetchFromGitHub {
+    owner = "fabiocolacio";
+    repo = "Marker";
+    rev = "${version}";
+    fetchSubmodules = true;
+    sha256 = "1iy7izyprf050bix8am1krqivgyxnhx3jm775v8f80cgbqxy7m5r";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    gtk3
+    gtksourceview
+    gtkspell3
+    webkitgtk
+    pandoc
+  ];
+
+  mesonFlags = [
+    "--prefix=${placeholder "out"}"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://fabiocolacio.github.io/Marker/";
+    description = "Markdown editor for the Linux desktop";
+    maintainers = with maintainers; [ trepetti ];
+    license = licenses.gpl3Only;
```

```suggestion
    license = licenses.gpl3Plus;
```
Might want to ask upstream as to whether this is GPL or LGPL - https://github.com/fabiocolacio/Marker/blob/2020.04.04/src/marker.c#L6-L9 says LGPL3+ but the toplevel LICENSE.md is standard GPL.

---

I just opened an [issue](https://github.com/fabiocolacio/Marker/issues/335) and will get to the bottom of this. I didn't actually see any reference to the LGPL in the sources or license file itself, but agree that it is not clear if it is GPL3 or GPL3+.

---

I think the 3+ part is clear, the source files all appear to all say GNU Library General Public License version 3 or later. My confusion is just that the toplevel LICENSE.md is the regular (non-library) type. Thanks for opening the issue, will stay tuned